_clanky/využití-data-50-část-1.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+detail: true
+title: Využití otevřených dat Zeměměřického úřadu pro povolování dočasných staveb - část 1
+ref: data50-1
+lang: cs
+image: ../attachments/články/využití-data-50/screen_vysledku.webp
+author: Michal Med
+date: 2021-01-18 03:14:15
+---
+První ze série článků o využití otevřených dat Zeměměřického úřadu se zabývá hledáním problému, který by data pomohla vyřešit a výběrem vhodných datových sad.
```

Comment:
podle mě by měla za "vyřešit" být čárka, protože je to věta vložená: ... článek ... se zabývá hledáním problému ... a výběrem vhodných datových sad